src/functions/uploadToRSIS/application/result-client.ts

```diff
@@ -155,3 +162,16 @@ const mapHTTPErrorToDomainError = (err: AxiosError): Error => {
   // Failed to setup the request
   return new Error(err.message);
 };
+
+const isRecordAutosaved = (test: StandardCarTestCATBSchema): BooleanAsNumber => {
+  // If the following properties are provided, we safely assume this is not an autosaved record.
+  // As testSummary maybe undefined, we use the non-null assertion operator.
+  return (
+    _.get(test, 'testSummary.additionalInformation', false) &&
+    _.get(test, 'testSummary.candidateDescription', false) &&
+    _.get(test, 'testSummary.identification', false) &&
+    _.get(test, 'testSummary.independentDriving', false) &&
+    _.get(test, 'testSummary.routeNumber', false) &&
+    _.get(test, 'testSummary.weatherConditions', false))
```

Comment:
Normally we import the specific thing we need for lodash e.g import { get } from 'lodash';
and use the get directly. So no need for the ugly underscore dot.

Think this should be done the same way for consistency.